weasyl/spam_filtering.py

```diff
@@ -76,7 +78,16 @@ def check(
         "is_test": _IS_ENVIRONMENT_TESTING,
     }
     if FILTERING_ENABLED:
-        return _akismet.check(**payload)
+        try:
+            return _akismet.check(**payload)
+        except ConnectionError as errmsg:
+            # Don't fail just because of a connection issue to the Akismet backend.
+            request = get_current_request()
+            request.environ['raven.captureMessage']("""
```

Comment:
The only thing I am curious about is if this will generate a single event that stacks, or if since the string content is dynamic, if this will spill out into multiple events. Reference ``WEASYL-9R`` where we track [here](https://github.com/Weasyl/weasyl/blob/a7329677411af5f3296bf58e5e5d8f3580be8858/weasyl/commishinfo.py#L79) that the rates have been updated.